sormas-api/src/main/resources/captions.properties

```diff
@@ -1870,6 +1870,8 @@ Sample.uuid=Sample ID
 Sample.samplePurpose=Purpose of the Sample
 Sample.samplingReason=Reason for sampling/testing
 Sample.samplingReasonDetails=Sampling reason details
+Sample.shipment=Shipment Status
+Sample.name=Samples
```

Comment:
Please use a generic caption for this, e.g. `samplesSamples` (and move it to the section with the other properties starting with samples above) since this is not a property of the entity